forge-gui/res/cardsfolder/upcoming/lithomantic_barrage.txt

```diff
@@ -0,0 +1,12 @@
+Name:Lithomantic Barrage
+ManaCost:R
+Types:Sorcery
+K:This spell can't be countered.
+A:SP$ Pump | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | IsCurse$ True | RememberObjects$ Targeted | SubAbility$ DBDmg | SpellDescription$ CARDNAME deals 1 damage to target creature or planeswalker. It deals 5 damage instead if that target is white and/or blue.
+SVar:DBDmg:DB$ DealDamage | Defined$ Remembered | NumDmg$ 1 | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | SubAbility$ DBDmgWU | StackDescription$ None
```

Comment:
too much
one line should be all you need: use SVar for Dmg with Compare or something

Reply:
Getting closer, but still no need to remember
Use `Targeted$Valid`

Reply:
You really dont need the Pump